test/cbor_serialize_test.c

```diff
@@ -295,6 +295,19 @@ static void test_auto_serialize(void **_CBOR_UNUSED(_state)) {
   _CBOR_FREE(output);
 }
 
+static void test_auto_serialize_no_size(void **_CBOR_UNUSED(_state)) {
+  cbor_item_t *item = cbor_new_definite_array(4);
```

Comment:
Please pick a some trivial item instead to make the test clearer, the for loop is distracting.

Reply:
Gotcha, done.